cypress/support/step-definitions/select-steps.js

```diff
@@ -220,3 +224,22 @@ Then("Select input has {string} value", (text) => {
 Then("Select input has no value", () => {
   getDataElementByValue("input").should("have.attr", "value", "");
 });
+
+Then("Option list has multiColumns header", () => {
+  multiColumnsSelectListHeader().should("have.length", "3").and("be.visible");
+});
+
+Then("Option list has multiColumns body", () => {
+  multiColumnsSelectListBody().should("have.length", "3").and("be.visible");
+});
+
+Then("The matching string is underline and bolded", () => {
```

Comment:
we need to pass here a `string` and be sure that the `string` we provided is only `underlined` and `bold`